src/Microsoft.Health.Dicom.SqlServer/Features/Query/SqlQueryGenerator.cs

```diff
@@ -234,6 +234,30 @@ public override void Visit(StringSingleValueMatchCondition stringSingleValueMatc
                 .AppendLine();
         }
 
+        public override void Visit(DoubleSingleValueMatchCondition doubleSingleValueMatchCondition)
+        {
+            var dicomTagSqlEntry = DicomTagSqlEntry.GetDicomTagSqlEntry(doubleSingleValueMatchCondition.DicomTag);
+            var tableAlias = GetTableAlias(dicomTagSqlEntry);
+            _stringBuilder
```

Comment:
does this pattern also work for custom tag? should tagKey be included in query as well?